### PR TITLE
Deliver fixes for Intelligent Management Issue 195

### PR DESCRIPTION
**Pull request by gmarcy**
https://github.ibm.com/liber8/squad-intelligentmanagement/issues/195

### CODE REVIEW

**Comment by gmarcy** [comment]
#build

File: dev/com.ibm.ws.microprofile.opentracing/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Import-Package: *
+
+Export-Package: \
+    com.ibm.ws.microprofile.opentracing, \
+    com.ibm.ws.microprofile.opentracing.cdi
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+-buildpath: \
+	io.opentracing:opentracing-api;version=0.30.0, \
```
Comment (ewhoch) [inline dev/com.ibm.ws.microprofile.opentracing/bnd.bnd]: is this just a re-add from a previously removed file? is it correct to have this depend directly on io.opentracing and not have it wrapped in something?

**Comment by ewhoch** [comment]
Looks like the associated build has some issues, so not ready to merge yet?

File: dev/com.ibm.ws.microprofile.opentracing/bnd.bnd (hunk 1 above)
Comment (gmarcy) [reply dev/com.ibm.ws.microprofile.opentracing/bnd.bnd]: it's doing the same thing that the com.ibm.ws.opentracing bundle is doing, so if it's wrong it's now wrong in two places.  will ask folks that have working on the code previously why that was.

**Comment by gmarcy** [comment]
#build

**Comment by gmarcy** [comment]
#build

**Pull request merged**